test/testenv.py

```diff
@@ -15,15 +15,10 @@
 
 def get_testid(testcase):
     """Convert a testcase filename into a test case identifier."""
-    name = os.path.splitext(os.path.basename(testcase))[0]
+    name = os.path.splitext(os.path.basename(testcase.filename))[0]
 
     return name
 
-def get_testcases(path):
-    for f in sorted(os.listdir(path)):
-        if f.endswith(testext):
-            yield os.path.join(path, f)
-
 options_schema = strictyaml.Map({
```

Comment:
This is missing a leading underscore, which is old news, but it could arguably be moved inside the class as well, which would make a nice pretext to rename.

Reply:
Pushed a commit fixing this.